README.md

```diff
@@ -8,6 +8,16 @@ cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release ..
 make
 ```
 
+## Build + install
+You can build for install using any path. If you install inside GStreamer plugin path there is no need to set `GST_PLUGIN_PATH` variable.
+
+```
+mkdir build && cd build
+cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/lib/x86_64-linux-gnu/gstreamer-1.0/cefsrc/ ..
+make
+make install
+```
+
```

Comment:
Just for someone who hasn't experienced this issue I would add a `Troubleshooting` section that goes something like this
## Troubleshooting

#### Error: `GPU process isn't usable. Goodbye.`

The subprocess bin path is configured with the absolute path when building. Then that's called at runtime to spawn the subprocess or alike, so no matter where the other files are it will spawn that bin from build path only. 

To solve this use `-DCMAKE_INSTALL_PREFIX` as stated above.

More #28